napari/layers/_labels_layer/model.py

```diff
@@ -559,7 +559,8 @@ def get_label(self, position, indices):
         int_coord[0] = int(round(coord[0]))
         int_coord[1] = int(round(coord[1]))
         label = self._slice_image(int_coord, image=self._raw_image)
-        return coord, label
+
+        return coord[:len(self.shape)], label
```

Comment:
why `len(self.shape)` instead of `self.image.ndim`?

Reply:
ah that would probably be even better! I just always forget about that

Reply:
will fix now

Reply:
fixed